pokete_classes/input.py

```diff
@@ -82,20 +85,17 @@ def ask_text(_map, infotext, introtext, text, name, max_len):
 
 
 def ask_ok(_map, text):
-    """Asks the player to an OK question
+    """Shows the player some information
     ARGS:
         _map: The map the question is asked on
         text: The question it self"""
     assert len(text) >= 4, "Text has to be longer then 4 characters!"
     text_len = sorted([len(i) for i in text.split('\n')])[-1]
-    with InfoBox(f"{text}\n{round(text_len / 2 - 2) * ' '}[O]k", name="Info",
+    with InfoBox(f"{text}\n{round(text_len / 2 - 2) * ' '} Ok ", name="Info",
```

Comment:
```suggestion
    with InfoBox(f"{text}\n{round(text_len / 2 - 2) * ' '} [O]k ", name="Info",
```

I would keep this, to be in one line with `ask_bool`.